src/java.base/share/classes/java/lang/invoke/X-VarHandleSegmentView.java.template

```diff
@@ -128,18 +128,18 @@ final class VarHandleSegmentAs$Type$s extends VarHandleSegmentViewBase {
         VarHandleSegmentViewBase handle = (VarHandleSegmentViewBase)ob;
         AbstractMemorySegmentImpl bb = checkAddress(obb, base, handle.length, true);
 #if[floatingPoint]
-        $rawType$ rawValue = SCOPED_MEMORY_ACCESS.get$RawType$Unaligned(bb.sessionImpl(),
+        $rawType$ rawValue = SCOPED_MEMORY_ACCESS.get$RawType$Unaligned(bb.session(),
```

Comment:
For instance, it's not clear to me why `baseSession()` is not called here.

Reply:
It seems that, if `get$RawType$Unaligned` calls `checkValidStateRaw()` this will end up throwing an exception